hack/install-linter.sh

```diff
@@ -1,21 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
Can we keep this file? This is how I install the linter on my laptop so that I can use `hack/module.sh lint` and be sure that I am using the same linter that CI is using.

Reply:
`hack/module.sh lint_in_docker` was meant to replace this, so we don't have to worry about installing anything.

I wonder if we could add a version parameter such as `hack/module.sh lint_in_docker va.b.c` to run a specific version of `golangci-lint`, that would imply we don't need a hard-coded lint version in this repo.

Reply:
file restored.